docs/contributing/documentation_guidelines.md

```diff
@@ -84,7 +84,7 @@ Make sure to include the correct language code for syntax highlighting, and to f
 
 ### Links
 
-Links are fundamental in documentation. Use internal links to tie your content to other docs, and external links as needed. Pick relevant text for links, so readers know where they will land.
+Links are fundamental in the documentation. Use internal links to tie your content to other docs, and external links as needed. Pick relevant text for links, so readers know where they will land.
```

Comment:
```suggestion
Links are fundamental in documentation. Use internal links to tie your content to other docs, and external links as needed. Pick relevant text for links, so readers know where they will land.
```

Also feels more correct to me without the article. It’s "documentation" in a general sense.